src/dotnet/commands/dotnet-tool/run/xlf/LocalizableStrings.zh-Hant.xlf

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="zh-Hant" original="../LocalizableStrings.resx">
+    <body>
+      <trans-unit id="CannotFindCommandName">
+        <source>Cannot find a command with command name '{0}'.</source>
+        <target state="new">Cannot find a command with command name '{0}'.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="CommandDescription">
+        <source>Run local tools installed or restored.</source>
+        <target state="new">Run local tools installed or restored.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="CommandNameArgumentName">
+        <source>COMMAND_NAME</source>
+        <target state="new">COMMAND_NAME</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="CommandNameColumn">
+        <source>command name of the tool to run.</source>
```

Comment:
Should this be upper case Command?

Reply:
Or `The command name of the tool to run.`, assuming this string is used somewhere (I couldn't find usage, but PRs already the best at searching for context).

Reply:
Yes are right, it is not used. I removed it.